src/display.rs

```diff
@@ -275,13 +275,19 @@ impl Display {
     ///
     /// This call may block if vsync is enabled
     pub fn draw(&mut self, mut terminal: MutexGuard<Term>, config: &Config, selection: Option<&Selection>) {
+        let bell_completed = terminal.visual_bell.completed();
+
         // Clear dirty flag
-        terminal.dirty = !terminal.visual_bell.completed();
+        terminal.dirty = !bell_completed;
 
         if let Some(title) = terminal.get_next_title() {
             self.window.set_title(&title);
         }
 
+        if !bell_completed {
```

Comment:
I think `bell_completed` is generally going to be false. It seems like this would raise the flag on pretty much every draw as a result.

Instead of using that indicator, would it be more correct if we just set a flag on `Term` when the bell is rang and clear the flag when setting urgency?